OpenSim/Simulation/Manager/Manager.h

```diff
@@ -208,9 +218,11 @@ class OSIMSIMULATION_API Manager
     //--------------------------------------------------------------------------
     // EXECUTION
     //--------------------------------------------------------------------------
+    bool integrate(SimTK::State& s, double finalTime);
```

Comment:
Ah I see...it's in the cpp file. I think the comment from the cpp file should be moved here.